test/fixtures/sections/headerimage.json

```diff
@@ -1,50 +1,48 @@
-[
```

Comment:
Just unwrapping the array since we unwrap sole sections directly on the mdast

(same for rest of the sections fixtures below)